Mlem/App/Views/Pages/Instance/InstanceDetailsView.swift

```diff
@@ -42,6 +53,40 @@ struct InstanceDetailsView: View {
                 ActiveUserCountView(activeUserCount: activeUserCount)
             }
             
+            FormSection {
+                VStack(spacing: Constants.main.standardSpacing) {
+                    HStack {
+                        Text("Uptime")
+                        
+                        Spacer()
+                        
+                        if case let .success(uptimeData) = uptimeData {
+                            NavigationLink(.instanceUptime(instance: instance, uptimeData: uptimeData)) {
+                                (Text("Details ") + Text(Image(icon: .general.forward)))
```

Comment:
*nit*

To avoid localizing the space

```suggestion
                                (Text("Details") + Text(verbatim: " ") + Text(Image(icon: .general.forward)))
```